app/views/content/early-years-teaching-training.md

```diff
@@ -19,74 +19,58 @@ date: "2021-03-29"
 description: |-
   Discover how to get Early Years Teacher Status (EYTS) and help to educate and inspire future generations. Official guidance on early years teacher training.
 calls_to_action:
-  EY-undergraduate:
-    name: feature_table
-    arguments:
-      - Cost: "Up to £9,250"
-        Salary: "No"
-        Funding available: "Yes"
-        Length of training: "3 or 4 years"
-        Qualification: "Early years teacher status"
-      - "Undergraduate early years teacher training"
   EY-graduate:
     name: feature_table
     arguments:
-      - Cost: "£7,000"
+      - Cost: "£7000 course fees are covered by the Department for Education and are paid directly to the training provider. You do not need to do anything to access this funding."
```

Comment:
```suggestion
      - Cost: "£7,000 course fees are covered by the Department for Education and are paid directly to the training provider. You do not need to do anything to access this funding."
```

Reply:
If the fees are covered by DfE, do we need to tell candidates how much they are? (Maybe yes if we think this would help encourage people by showing how much they are 'saving' / being valued?)

If we think that we do need to specify the amount, should it be upfront like this? If I was scanning this as a user, is there a danger that I would focus on the £7k and miss the fact that DfE pays for it?

Reply:
Good point, I was thinking it would help encourage as you describe but maybe need to think again about the ordering.